asc2018/vdr/electrical/WaterlooMidnightSun_ASC2018VDR_BatteryTechReport.tex

```diff
@@ -0,0 +1,119 @@
+% LuaLaTeX
+\pdfvariable minorversion=7
+\documentclass[10pt]{article}
+
+% Packages
+\usepackage[no-math]{fontspec}
+\usepackage{geometry}
+\usepackage{hyperref}
+\usepackage[backend=biber]{biblatex} % we use the biber implementation of biblatex for bibliographies
+%\usepackage[nonumberlist, toc]{glossaries}
+\usepackage{graphicx}
+\usepackage{pdfpages}
+\usepackage{booktabs}
+\usepackage{float}
+\usepackage{subcaption}
+\usepackage{fancyhdr}
+\usepackage{amsmath}
+\usepackage{siunitx}
+\usepackage{multirow}
+
+% Formatting
+\geometry{letterpaper, portrait, margin=.85in}
+\defaultfontfeatures{Ligatures=TeX}
+\setmainfont[
+    BoldFont       = Avenir Medium,
+    ItalicFont     = Avenir Book Oblique,
+    BoldItalicFont = Avenir Medium Oblique
+]{Avenir Book}
```

Comment:
Not a blocker on merging but this font isn't available anywhere so far as I can find. Why are we making this hard to work on? We should at least document where to get it in a `README.md`.

Reply:
I basically copied Minghao's original template and whatever fonts he had originally set. I can link to where I found the fonts, or add them to the Drive?

Reply:
Added the fonts to `Electrical > Documentation > Fonts > Avenir Book` on the drive since I don't have access to whatever the parent of `Electrical` is.